test/integration/indexSpec.js

```diff
@@ -15,6 +16,7 @@ describe('All with queries', () => {
     before((done) => {
         sqliteUtil.createSQLiteDatabase(done);
     });
+    after(done => {stop(done)});
```

Comment:
Would simply `after(stop)` work? If not, convert to a three-line form (like the previous three lines) for general readability, and while you're at it add the missing semicolon.